File: src/main/python/codeeval/easy/RealFake/real_fake.py
import sys
with open(sys.argv[1], 'r') as lines:
    for line in lines:
        numbers = [ int(ch) for ch in line.replace(' ', '').strip() ]
        sum     = 0
        for i, x in enumerate(numbers):
            sum += 2 * x if i % 2 == 0 else x
        print("Real" if sum % 10 == 0 else "Fake")

File: src/main/scala/codeeval/easy/RealFake/RealFake.scala
package codeeval.easy.RealFake

object Main extends App {
    scala.io.Source.fromFile(args(0))
                   .getLines
                   .map(_.replace(" ", "").map(_.asDigit).zipWithIndex)
                   .map(_.map { case (x, i) => if (i % 2 == 0) 2 * x else x })
                   .map(numbers => if (numbers.sum % 10 == 0) "Real" else "Fake")
                   .foreach(println)
}